shared/wallets/send-form/participants/to-field.js

```diff
@@ -0,0 +1,200 @@
+// @flow
+import * as React from 'react'
+import * as Kb from '../../../common-adapters'
+import * as Styles from '../../../styles'
+import {ParticipantsRow} from '../../common'
+import {SelectedEntry, DropdownEntry, DropdownText} from './dropdown'
+import type {Account} from '.'
+import type {CounterpartyType} from '../../../constants/types/wallets'
+
+type ToFieldProps = {|
+  recipientType: CounterpartyType,
+  /* Used for send to stellar address */
+  incorrect?: string,
+  onChangeAddress?: string => void,
+  /** Used for sending from account to account */
+  accounts: Account[],
+  onChangeSelectedAccount: (accountName: string) => void,
+  onLinkAccount?: () => void,
+  onCreateNewAccount?: () => void,
+  /* Used to display a keybase profile */
+  username?: string,
+  fullName?: string,
+  onShowProfile?: string => void,
+  onRemoveProfile?: () => void,
+|}
+
+type ToFieldState = {|
+  selectedAccount: ?Account,
+|}
+
+class ToField extends React.Component<ToFieldProps, ToFieldState> {
+  state = {
+    selectedAccount: null,
+  }
+
+  onDropdownChange = (node: React.Node) => {
+    if (React.isValidElement(node)) {
```

Comment:
If any flow wizards know a better way to achieve better type-checking here, please let me know